src/gup.rc

```diff
@@ -82,3 +82,16 @@ BEGIN
     PUSHBUTTON   "No",IDNO,110,60,50,14
     PUSHBUTTON   "Never",IDCANCEL,170,60,50,14
 END
+
+IDD_UPDATE_DLG DIALOGEX 0, 0, 187, 88
+STYLE DS_SETFONT | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
+EXSTYLE WS_EX_TOOLWINDOW | WS_EX_WINDOWEDGE
+CAPTION "Notepad++ Update"
+FONT 8, "MS Shell Dlg", 400, 0, 0x1
+BEGIN
+    LTEXT           "No update is available.", IDC_UPDATE_STATIC1, 56, 6, 74, 8
+    LTEXT           "The auto-update may not have been activated yet.", IDC_UPDATE_STATIC2, 10, 24, 166, 8
+    CONTROL         "<a id=""id_download"">Go to the download page</a>", IDC_DOWNLOAD_LINK,

```

Comment:
Can "Go to the download page" be translatable?
It's quite awkward some are not translated.

Reply:
Done. However, I've noticed that localization is not actually used in WinGUP inside Notepad++.

Reply:
I'll bind the language used in Notepad++ to WinGUp:
With 2 options:
1. WinGUp can be called with the same argument of Notepad++ `-L : Apply indicated localization, langCode is browser language code` but in this case, all the language xml should be installed.
2. Same "nativeLang.xml" as in Notepad++ can be done, so it could be install only one file which is the same chosen language.

Of course, I have to ask translators adding the translation for WinGUp.

Reply:
Option 2 seems easier in terms of maintaining - all text in one place. 
Maybe, this way it will be easier for translators as well.